From 9e305e35900055ebf090b772b774ead099e6c0e5 Mon Sep 17 00:00:00 2001
From: sanshan zhang <sanshan.zhang@nxp.com>
Date: Wed, 11 Jan 2017 15:05:30 +0800
Subject: [PATCH 246/296] Fix issue when start
 android/com.android.internal.backup.LocalTransportService

When SOC vendor do not install gapps, it will not have permission
backup-transport-whitelisted-service which define in gapps/etc/sysconfig/google.xml.

This patch add the backup-transport-whitelisted-service in
system/etc/permissions/platform.xml

Error log when boot up:
 W BackupManagerService: Proposed
transport ComponentInfo{android/com.android.internal.backup.LocalTransportService}
not whitelisted; ignoring
---
 data/etc/platform.xml | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/data/etc/platform.xml b/data/etc/platform.xml
index def91fcd340..6582c7ced52 100644
--- a/data/etc/platform.xml
+++ b/data/etc/platform.xml
@@ -191,6 +191,9 @@
     <!-- These are the packages that are white-listed to be able to run as system user -->
     <system-user-whitelisted-app package="com.android.settings" />
 
+    <!-- These are the transport services that are white-listed to be able to run -->
+    <backup-transport-whitelisted-service service="android/com.android.internal.backup.LocalTransportService" />
+
     <!-- These are the packages that shouldn't run as system user -->
     <system-user-blacklisted-app package="com.android.wallpaper.livepicker" />
 </permissions>
-- 
2.11.0

